Edits to the remaining index pages as part of site review (#205)
pinot-contrib/pinot-docs: basics/indexing/geospatial-support.md

![Hexagonal grid in H3](../../.gitbook/assets/geoindex-h3.png)

### How to use geoindex

> To use the geoindex, first declare the geolocation field as bytes in the schema, as in the example of the [QuickStart example](https://github.com/apache/pinot/blob/master/pinot-tools/src/main/resources/examples/batch/starbucksStores/starbucksStores\_schema.json#L25).

Reviewer: link 404s
Didn't see "quickstart example" in starbucksStores directory...maybe [this file](https://github.com/apache/pinot/blob/master/pinot-tools/src/main/resources/examples/batch/starbucksStores/starbucksStores_schema.json)?
....but  doesn't look like file specifically refers to "geolocation" field anywhere in the schema...unless it's `location_st_point`? 

@xiangfu0 can you help us? thank you!

Author: I have fixed the 404. I am going to merge so as not to delay all the other improvements, but would love to see an answer to @kelseiv 's question.

limit 1000
```

### How geoindex works

> Geoindex in Pinot accelerates the query evaluation without compromising the correctness of the query result. Currently, geoindex supports the `ST_Distance` function used in the range predicates in the `WHERE` clause, as shown in the query example in the previous section.

Reviewer: ```suggestion
The Pinot geoindex accelerates query evaluation while maintaining accuracy. Currently, geoindex supports the `ST_Distance` function in the `WHERE` clause.
```

Reviewer: As a practice, let's omit "as shown in the previous section," and do the following instead:
-  Refer directly to name of example 
-  Make sure conceptual content (how geoindex works) comes before procedural (how to use geoindex)

> As in the example diagram above, if we want to find all relevant points within a given distance at San Francisco (represented in the area within the red circle), then the algorithm with geoindex works as the following:

Reviewer: ```suggestion
As in the example diagram above, if we want to find all relevant points within a given distance around San Francisco (area within the red circle), the geoindex works as follows:
```

Reviewer: the verbiage doesn't line up between the introductory clause and the bullets that follow--what is the user doing versus what is the geoindex doing?

> * Find the H3 distance `x` that contains the range (such as, within a red circle)

Reviewer: ```suggestion
* Find the H3 distance `x` that contains the range (for example, within a circle)
```
